azsettings/settings.go

```diff
@@ -16,8 +16,9 @@ type AzureSettings struct {
 	WorkloadIdentityEnabled  bool
 	WorkloadIdentitySettings *WorkloadIdentitySettings
 
-	UserIdentityEnabled       bool
-	UserIdentityTokenEndpoint *TokenEndpointSettings
+	UserIdentityEnabled            bool
+	UserIdentityTokenEndpoint      *TokenEndpointSettings
+	UserIdentityServiceCredentials bool
```

Comment:
Let's think about the property name...

Reply:
How about `UserIdentityFallbackCredentialsEnabled`?